modular_citadel/code/modules/projectiles/guns/pumpenergy.dm

```diff
@@ -173,7 +179,7 @@
 	projectile_type = /obj/item/projectile/energy/electrode/pump
 	select_name = "stun"
 	fire_sound = 'modular_citadel/sound/weapons/LaserSlugv3.ogg'
-	e_cost = 300
+	e_cost = 400
```

Comment:
Just three shots? That's pretty meager considering the particle defender is meant primarily for riot control